samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<p>&nbsp;it can adapt using just a few examples and a prompt</p><p> </p><p>Reformulate it to: &nbsp;it can infer/understand the task using just a few examples and a prompt</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>